kinto/plugins/default_bucket/__init__.py

```diff
@@ -73,8 +80,14 @@ def create_collection(request, bucket_id):
             'description': "Invalid collection_id id"
         }
         raise_invalid(request, **error_details)
+    data = {'id': collection_id}
     try:
-        collection = resource.model.create_record({'id': collection_id})
+        collection = resource.model.create_record(data)
+        # Since the current request is not a resource (but a straight Service),
+        # we simulate a request on a bucket resource.
+        # This will be used in the resource event payload.
+        request.current_resource_name = 'collection'
+        resource.postprocess(data, action=ACTIONS.CREATE)
```

Comment:
I see twice the same code here in this file, so having a helper for it should make it easier to maintain.